README.md

```diff
@@ -134,6 +134,8 @@ You can change the initial options passing props to the component.
 
 **showOverlay** {bool}: Display an overlay with holes above your steps (for tours only). Defaults to `true`
 
+**allowClicksThruHole** {bool}: Allow mouse clicks and touch events within overlay hole, and prevent `hole:click` callback from being sent.  Defaults to `false`
```

Comment:
I think is better to use `mouse and touch events` instead of just mention clicks.